pysumma/option.py

```diff
@@ -12,7 +12,7 @@ def __init__(self, name, value=None):
         self.name = name
```

Comment:
I have a question. What does `optiontype` mean in OptionContainer class?  31 line of option.py

Reply:
`optiontype` is the type of option. All options are subclasses of the `BaseOption` which is defined in the same file. For example, in `decisions.py` we define `DecisionOption` which specifies what a single option in the decisions file should look like.